index.html

```diff
@@ -118,10 +118,11 @@ <h2 class="contactme-title">Contact me</h2>
                 <p class="contactme-detail">If you have an application you are interested in developing, a feature that
                     you need built or a
                     project that needs coding. I’d love to help with it</p>
-                <form action="https://formspree.io/f/mzbwzepn" method="post">
+                <form id="signup" action="https://formspree.io/f/mzbwzepn" method="post">
                     <fieldset class="finalforms">
                         <input name="name" type="text" maxlength="30" placeholder="Alejandra" required>
-                        <input name="email" type="email" placeholder="alejandrakeber@gmail.com" required>
+                        <input id="email" name="email" type="email" placeholder="alejandrakeber@gmail.com" required>
+                        <p id="error"></p>
```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 
- [ ] as the stated in the requirements. 
- [ ] If the validation is not OK, you show an error message to the user `near the submit button` informing them of the error and the form is not sent.
- [ ] kindly move the error message near the submit button. 🥂